modules/etcd-cert-alerts-metrics-signer.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: CONCEPT    
+[id="etcd-cert-alerts-metrics-signer_{context}"]
+== etcd certificate rotation alerts and metrics signer certificates
+
+Two alert types inform users about pending `etcd` certificate expiration:
+[horizontal]
+`etcdSignerCAExpirationWarning`:: Occurs 730 days until the signer expires.
+`etcdSignerCAExpirationCritical`:: Occurs 365 days until the signer expires.
+
+You can rotate the certificate for the following reasons:
+
+* You receive an expiration alert
+* The private key is leaked
```

Comment:
Per ISG: If the list items comprise only complete sentences, include a period after each sentence.
```suggestion
* You receive an expiration alert.
* The private key is leaked.
```